corehq/apps/accounting/templates/accounting/partials/renew_plan_selection.html

```diff
@@ -0,0 +1,44 @@
+{% load i18n %}
```

Comment:
This django template is duplicative of most of the stuff in `accounting/partials/confirm_plan_summary.html` but that one is also used elsewhere in `domain/confirm_plan.html` so I didn't really want to modify it directly.